pushy/src/main/java/com/turo/pushy/apns/util/ApnsPayloadBuilder.java

```diff
@@ -530,6 +535,47 @@ public ApnsPayloadBuilder setThreadId(final String threadId) {
         return this;
     }
 
+    /**
+     * <p>Sets the summary argument for this notification. The summary argument is:</p>
+     *
+     * <blockquote>The argument to be inserted in the summary for this notification when the system groups the
```

Comment:
If this is a quote, can we cite the source (perhaps in a `@see` tag)? If it's not a quote, I think we can just use a normal paragraph (`<p>...</p>`).

Reply:
Updated comments in ed6aed6.